app/models/manageiq/providers/nutanix/infra_manager/vm.rb

```diff
@@ -1,4 +1,14 @@
 class ManageIQ::Providers::Nutanix::InfraManager::Vm < ManageIQ::Providers::InfraManager::Vm
+  include ManageIQ::Providers::Nutanix::InfraManager::Vm::Operations::Power
+  # Completely custom logic - no host/storage check
+  supports :start do
+    unsupported_reason_add(:start, _('The VM is already powered on')) if raw_power_state.to_s.downcase == 'on'
+  end
+
+  supports :stop do
+    unsupported_reason_add(:stop, _('The VM is already powered off')) if raw_power_state.to_s.downcase == 'off'
+  end
```

Comment:
Now that we have storage/host parsing do we need to override this still?  I'd rather fix the parser to work with the generic start/stop supports if possible.